pkg/policy/api/selector.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	"github.com/cilium/cilium/pkg/labels"
 
+	"github.com/mitchellh/hashstructure"
```

Comment:
Was this imported into `vendor/`?

Reply:
I believe so, here's the commit which updates vendor.conf and imports the code:
https://github.com/cilium/cilium/pull/1599/commits/1ae6a2c2124c385c9257f006eaafc8ca414d5898